src/common.c

```diff
@@ -49,6 +49,7 @@
 #include <inttypes.h>
 #include <netinet/in.h>
 #include <arpa/inet.h>
+#include <infiniband/ib.h>
```

Comment:
This is a linux specific file, and likely requires libibverbs to be installed.

Reply:
Hmm. That's needed for referencing "struct sockaddr_ib". I think we can drop this patch.

Reply:
I think the verbs provider may need its own addr_size call that calls the common call, but adds support for AF_IB.